src/control/SConscript

```diff
@@ -187,7 +187,8 @@ def scons():
     # Explicitly link RTE & SPDK libs for CGO access.
     senv.AppendENVPath("CGO_LDFLAGS", cgolibdirs + \
                        " -lspdk_env_dpdk -lspdk_thread -lspdk_bdev" + \
-                       " -lspdk_copy -lrte_mempool  -lrte_mempool_ring" + \
+                       " -lspdk_copy -lrte_vhost -lrte_mempool" + \
```

Comment:
Why rte_vhost is required now?

Reply:
something changed in dependency tree between versions, we get undefined symbol messages . if we don't include it here. nothing in the code has changed to require it so presumably it's now depended upon by one of the other libraries we are linking.  
I could look into finding a way to remove it if you think it's a priority.

Reply:
I'm confused as to why the change is not required in bio/SConstript though , given we are linking with the same libraries